README.md

```diff
@@ -504,3 +487,29 @@ Failed requests get thrown as a `TrusonaException`, which contains a message abo
 ### Need additional help?
 
 Contact us at engineering@trusona.com
+
+
+
+## Development
```

Comment:
I would move this into its own file (DEVELOP.md). Keep the README just user facing documentation.